amplify/src/main/java/com/github/stkent/amplify/tracking/predicates/LastTimePredicate.java

```diff
@@ -40,4 +40,9 @@ public void eventTriggered(@NonNull IEvent event) {
             updateEventValue(event, currentTime);
         }
     }
+
+    @Override
+    public Long defaultValue() {
+        return 0L;
```

Comment:
Either one of those is going to have to have some awkward logic if we do that, right?